README.md

```diff
@@ -43,10 +43,18 @@ They will need to filtered/removed from the unit test run when running in defaul
 
 #### Automation of Above
 
-To automate the above strategy we can create a wrapper script for `clarinet test` which
-copies / filters the mining and stacking contracts and copies the `clarinet.toml.local` file to
-`clarinet.toml`. Running the tests in local mode will include the tests in the `tests/extensions/local`
-directory. Running them in integration mode will exclude these tests.
+To automate the above strategy;
+
+```bash
+scripts/copy-contracts.sh
+```
+
+To copy and filter the mining and stacking contracts and copies the `clarinet.local.toml` file to
+`clarinet.toml`. To make the script executable in your environment run
+
+```bash
+chmod 755 scripts/copy-contracts.sh
+```
```

Comment:
Moved this text to be with the command as a separate item.

```suggestion
If you are unable to execute the script, run the following command to update the permissions then try again:

```bash
chmod 755 scripts/copy-contracts.sh
```
```